dashboard/src/shared/schema.ts

```diff
@@ -6,15 +6,15 @@ import * as jsonpatch from "fast-json-patch";
 import * as yaml from "js-yaml";
 import { isEmpty, set } from "lodash";
 // TODO(agamez): check if we can replace this package by js-yaml or vice-versa
-import YAML from "yaml";
-import { nullOptions } from "yaml/types";
-import { Type } from "yaml/util";
+import YAML, { ToStringOptions, Scalar } from "yaml";
 import { IBasicFormParam } from "./types";
 
 const ajv = new Ajv({ strict: false });
 
-// Avoid to explicitly add "null" when an element is not defined
-nullOptions.nullStr = "";
+const toStringOptions: ToStringOptions = {
+  defaultStringType: Scalar.QUOTE_DOUBLE, // Preserving double quotes in scalars (see https://github.com/vmware-tanzu/kubeapps/issues/3621)
```

Comment:
It seems that the default value for `defaultKeyType` is null, perhaps it is good to explicitly add 
`defaultKeyType: 'PLAIN'` 
so that we cover both key and values?

Reply:
Good idea! Thanks!